battery

```diff
@@ -7,14 +7,15 @@ battery usage:
     -h, --help    print this message
     -t            output tmux status bar format
     -z            output zsh prompt format
+    -s            output screen prompt format
     -e            don't output the emoji
     -a            output ascii instead of spark
     -b            battery path            default: /sys/class/power_supply/BAT0
     -p            use pmset (more accurate)
-  colors:                                                 tmux     zsh
-    -g <color>    good battery level      default: 1;32 | green  | 64
-    -m <color>    middle battery level    default: 1;33 | yellow | 136
-    -w <color>    warn battery level      default: 0;31 | red    | 160
+  colors:                                                 tmux     zsh	 screen
+    -g <color>    good battery level      default: 1;32 | green  | 64	| g
+    -m <color>    middle battery level    default: 1;33 | yellow | 136	| y
+    -w <color>    warn battery level      default: 0;31 | red    | 160	| k
```

Comment:
I think the indentation got busted here, could you fix that?

Reply:
Specifically for the `| g`

Reply:
And I believe you wrote `k` instead of `r` for the `-w` option?

Reply:
Yes you are right. I tried retabbing the file but it retabs all of it, I’ll try manually fixing it